WordPress/WordPressUITests/Flows/LoginFlow.swift

```diff
@@ -4,7 +4,7 @@ class LoginFlow {
 
     @discardableResult
     static func login(email: String, password: String) -> MySiteScreen {
-        logoutIfNeeded()
+        try! logoutIfNeeded()
```

Comment:
I force tried (`try!`) here because I didn't want to keep traversing the call graph. In the "final" implementation, I'd like not to have any `try!`. The reason I'm saying this is because XCTest knows what to do when a `try` `throws` and report it as a test failure instead of a crash.

_Shameless plug: I wrote of such an instance [today](https://mokacoding.com/blog/how-to-test-async-await-code-in-swift/)._